lib/take_away_menu.rb

```diff
@@ -0,0 +1,32 @@
+class TakeAwayMenu
+attr_reader :items, :list_dishes
+
+def items
+  @items = @items = {'pasta' => 5, 'chips' => 2, 'salad' => 4, 'pizza' => 6, 'soup' => 5}
+end
+
+    def list_dishes
+       puts "1. pasta: £5.00"
```

Comment:
Could this method use the `@items` hash, rather than hardcoding the dishes and prices?